mspray/apps/main/views/performance.py

```diff
@@ -322,6 +324,7 @@ def get_context_data(self, **kwargs):
                 ).name,
                 "district": district,
                 "district_name": district.name,
+                "community_health_worker": community_health_worker,
```

Comment:
If we are using a "global context processor" is it still necessary to the change the individual context of a view in this way?  Isn't avoiding that why we chose a global context processor?